libs/aws/langchain_aws/chat_models/bedrock.py

```diff
@@ -363,6 +380,10 @@ def convert_messages_to_prompt(
                 human_prompt="\n\nUser:",
                 ai_prompt="\n\nBot:",
             )
+        elif provider == "deepseek":
+            prompt = convert_messages_to_prompt_deepseek(messages=messages)
+        elif provider == "qwen":
+            prompt = convert_messages_to_prompt_qwen(messages=messages)
```

Comment:
The different provider requirements for the Qwen distill vs. LLama-distill / R1 base are convoluted from a user standpoint. 

Is there any way we can determine the model given the SageMaker endpoint? If it's not possible, we should add a note to the ChatBedrock [docs](https://python.langchain.com/api_reference/aws/chat_models/langchain_aws.chat_models.bedrock.ChatBedrock.html#langchain_aws.chat_models.bedrock.ChatBedrock.provider) about provider usage with specific DeepSeek models.